tests/particle_test.cc

```diff
@@ -678,8 +711,6 @@ TEST_CASE("Particle is checked for 2D case", "[particle][2D]") {
     // TODO Assert: REQUIRE_NOTHROW(particle->compute_updated_position(dt,
     // true)); Compute volume
     // TODO Assert: REQUIRE(particle->compute_volume() == false);
-    // Update volume should fail
-    // TODO Assert: REQUIRE(particle->update_volume() == false);
```

Comment:
Why is this deleted?

Reply:
That's commented test. Tbh, there are many of them found all related to assert. I would prefer to clean them up. What do you think @kks32?

Reply:
I think catch has a `REQUIRE_THROW` with a specific exception, we could test against those